### DISCUSSION TITLE
Ban event

### DISCUSSION THREAD

**Discussion by masterYl** [sunflower-land/sunflower-land#1358] [Polls]
Hello, adam, something has happened in the sunflower community recently. We express strong dissatisfaction with some recent official actions. I don’t know if adam knows about this, or has recognized the result of this event?

This incident is: the recent indiscriminate and wrong account ban incident, and the discord operators did not do their due diligence, and does this behavior violate the principles of web 3.0 in the white paper?

Okay, let me sort out, from my perspective, the ins and outs of this event:

In April, I came into contact with the game Sunflower, and what attracted me most came from what Adam said in the white paper:
![image](https://user-images.githubusercontent.com/50456562/185853073-f1706abe-0d2f-42ed-ac07-4b4f5ac4ba11.png)

I believe that what you said about Sunflower is a web3 and community-driven game. Currently, what is the broadest definition of web3: decentralization, users have full rights to [view], [read and write], and [control] Internet assets .

I agree with this concept very much. I believe that you can develop Sunflower better, so I invested a lot of money in the early stage and discussed the future of Sunflower with you in the discord community, and hope that other players can also believe in Sunflower could do better

Do you know what i did？
    invested a lot of money
    In the discord community, let other players believe in Sunflower and reply to the official news
![下载 (1)](https://user-images.githubusercontent.com/50456562/185853652-a7f2ce4e-ff47-4517-ba03-208eac9aa351.png)
![下载 (3)](https://user-images.githubusercontent.com/50456562/185853692-f8b2bdb6-7190-4cf0-a3fb-0fd018c80d49.png)
    Free lots of wood for newcomers
![下载 (2)](https://user-images.githubusercontent.com/50456562/185853927-662ce01d-634d-4429-843d-ebe20395eff4.png)
    Pay my own money to have my family and friends play this game

This can fully prove that I am an absolutely rare sunflower loyal fan and investor
But recently, you guys banned my and my friend's accounts.
That morning, my mother suddenly took her banned account and asked me what was going on, and later found out that my own and all my friends' accounts were banned!
![下载 (5)](https://user-images.githubusercontent.com/50456562/185854190-5f005867-595a-4504-b76c-a850710a93a8.png)

I can prove that these accounts are not played by me, but by my friends. They are all officially bought land. Indeed, I invested in them and let them hold NFTs, but they are indeed real people playing, I do not I think this violates the multi-number rule:
You can look at the chat history, I recommend this web3.0 game for my family and friends, and now, they are closed!
![下载 (4)](https://user-images.githubusercontent.com/50456562/185854425-db421f0c-596b-4b66-8a73-88711749df62.png)
![下载](https://user-images.githubusercontent.com/50456562/185854438-5555c6e8-ff12-4cd0-9603-fd8107731f13.png)

I understand you very well: closing the robot account, but your operation this time has seriously affected the true sunflower fans. Let's 

See what you have done this time:
Sunflower resource structure:
![image](https://user-images.githubusercontent.com/50456562/185854565-dd25dabd-0532-4946-afc6-c97498d60aca.png)

What you are doing now:
![image](https://user-images.githubusercontent.com/50456562/185854633-dade1bc2-12ee-4579-ace9-2a0f0e1b923c.png)

So how should the sunflower work? You don't want to get the investment yourself, to expand the bigger team, and then kill the investor, is that what you want?

I reported this incident in the help channel, but it was not treated fairly, I am going to complain to one of your operating members: Labochi

I hold so many resources, I have hardly sold mine, and your Labochi, sold his land for the reason: to heal his friends

What? ? ? ? ? ? ? A core operating member who sold his land and then banned the land of a bunch of loyal investors? ? ? ? ?

Okay, so, since one person and multiple accounts are completely banned, what about the following account? Is this Labochi's? Ask him to come out and explain

FarmID: 7841
WalletAddr: 0x98F6723C38eD4d49a4A374a65B98Cd9D2d47F9c2
FarmAddr: 0xeE4D2D01c32B6484a73755576B919e7f4c9b22c1

Is this another account of his? Ask him to come out and clarify the truth!
Okay, that's about it, let's summarize:
①We can understand banning robot accounts, but it is wrong to ban non-robot accounts, which violates the web 3.0 project mentioned in the white paper, which is completely centralized hegemony
(I also have evidence to prove that the accounts I banned are all real players, one account per person!)

②There are still many innocent accounts involved in this incident. If similar incidents occur repeatedly, will it affect the confidence of other investors? How long can sunflowers live?

③Since the white paper says that it is driven by the community, does this collective resistance act violate the community? And for the conclusion of multiple accounts, should I open a post to see the opinions of the community?

④Should closed accounts be dealt with as bans? Or is it just closed for a few days and given a warning? Or do multiple accounts charge higher handling fees (even 99%) to deal with this problem

⑤Please Labochi come out and explain: the core operating member sold his land and held another account

**Comment by TheDcol** [comment]
Labochi sold his farm to someone else, this is allowed AFAIK, so he does not have to explain anything.

**Reply by masterYl** [reply]
Then how to explain his other account? If one person is strictly required to have one account?

**Reply by TheDcol** [reply]
1 person can only play 1 farm at a time

Can i get an invite to your community group?

**Reply by masterYl** [reply]
I'm not very active in the community anymore, if you have WeChat, I can invite you in

**Reply by TheDcol** [reply]
a community invite would be appreciated, would love to see all the people talking about sfl there!

**Reply by loxa1010** [reply]
Labochi ，He can choose not to explain

But he is a law enforcement officer, and he must be clean and unquestionable

So he has to present evidence that after he sold the land, where did the other land come from

This is not an official mint, if it is a transaction it must prove that the original holder is not him

Prove the player's doubts about him wrong

Otherwise, how to convince people, how to make people believe in Sunflower's team justice

Law enforcement officers must be impartial and not have ambiguous behavior

**Comment by 1176517584** [comment]
Hope to treat real players' demands, do not do their own, do not consider the feelings of the players

**Comment by loxa1010** [comment]
an admin who ditched the game
What qualifications are there to seal players?
they do not participate
How to know what players want

**Reply by enterfeda** [reply]
strange point of view

**Comment by qiongloo** [comment]
I think the project party is not humane enough, and the experience is very poor. If it is banned, it will be permanently banned, and it must be given a chance to reform. 2. The banning rules are not clear, especially in the handling of multiple accounts. At present, according to my observation, it is determined by the relationship on the chain. This is problematic, because transactions are casual, unless transactions are prohibited. The identification of multiple numbers should be the same as that of robots. After all, the data on the chain is created, and no one can tell. 3. The billing process is also very slow. I know that many people miss the event due to accidental ban and there is no compensation. I hope the project party will not hurt the hearts of fans.

**Comment by Aaronkoob** [comment]
Is masterye the same person in discord channel? I saw you arguing all the time a few days ago, so I also investigated it. It is obvious that you are lying and distorting the facts. I will upload some of my investigations later. I think there is no problem with the team's practices because you have violated the rules. You should not incite people to slander and frame anyone in the team
![971661155594_ pic](https://user-images.githubusercontent.com/111733372/185871644-4ebfd520-2c55-4579-ad88-281fa7963012.jpg)
![951661155592_ pic](https://user-images.githubusercontent.com/111733372/185871731-c5644dad-f979-493f-9b7f-d8193851d0ac.jpg)
![961661155593_ pic](https://user-images.githubusercontent.com/111733372/185871760-195f1ddd-b394-495f-b9e9-fa415ccea57b.jpg)

.

**Reply by masterYl** [reply]
As you can see, I gave them all my assets and I let them play, what's the problem? Then I didn't distort the truth, didn't slander your member, he sold his account, he has another account, it's a fact

**Reply by masterYl** [reply]
I speak my truth instead of inciting others to destroy the community, look at the HELP channel, how many people have encountered this, not just me

**Reply by 1176517584** [reply]
White list of users, project so blatant transfer assets, and administrator's malicious real player's farm, this is a joke, you said, who are you not to investigate the administrators in what, as the administrator is also buy and sell the farm, and then to a player is not recommended to buy, and myself are involved in buying and selling,  This is a joke to others. How can you produce an NFT by yourself without participating in the trade? Dream on.  Administrators can ignore the rules

**Reply by loxa1010** [reply]
> 這麼明顯的多賬號，太不可思議了，這人怎麼老是吹毛求疵

where do you have multiple accounts

Are you blind?

These nft assets can already be purchased, and mint's

Hold the majority of assets, invite family, friends, partners to play together

Let them have a better gaming experience, what's wrong

Does it only start from 0, there is no violation

Those NFT accounts that directly transfer NFT, non-OpenSea transactions

All should be investigated, all should be presented with evidence to prove innocence

**Comment by 17ss** [comment]
there are many internal farms here, 
they can snipe trade, multiple-account, bot scripts without being banned,
because they are privileged. so the rule is a joke, it has become a tool to exclude others.

Privilege FarmID: 26, 486, 100025, 144826
![image](https://user-images.githubusercontent.com/56461773/185871801-e650cacd-626c-4f95-be6e-910f5fddc889.png)

**Reply by masterYl** [reply]
What an exciting thing, the project side will come out and explain it

**Reply by 1176517584** [reply]
Whitelist users, the project side so blatant transfer assets, and the administrator also malicious seal real players farm, this is really a joke, laugh dead

**Reply by loxa1010** [reply]
The project party must explain

**Reply by DemonDG** [reply]
What happened to Farm 26 and Farm 486

**Reply by 1176517584** [reply]
> 农场 26 和农场 486 发生了什么

The management on Discord avoided talking about it and dared not confront it

**Reply by libbiashyra** [reply]
is this real?that's so crazy.

> there are many internal farms here, they can snipe trade, multiple-account, bot scripts without being banned, because they are privileged. so the rule is a joke, it has become a tool to exclude others.
> 
> Privilege FarmID: 26, 486, 100025, 144826 ![image](https://user-images.githubusercontent.com/56461773/185871801-e650cacd-626c-4f95-be6e-910f5fddc889.png)

**Reply by 17ss** [reply]
yes, it's real,please search "486" in official discord,pretty sure all those farms are linked,you will find it is a privileged farm

> is this real?that's so crazy.
> 
> > there are many internal farms here, they can snipe trade, multiple-account, bot scripts without being banned, because they are privileged. so the rule is a joke, it has become a tool to exclude others.
> > Privilege FarmID: 26, 486, 100025, 144826 ![image](https://user-images.githubusercontent.com/56461773/185871801-e650cacd-626c-4f95-be6e-910f5fddc889.png)

**Reply by masterYl** [reply]
Just now, I sent this screenshot in the help channel, and now, they deleted my account in discord, hahaha, they have not responded positively to this matter, and they have deleted me. So when will this post be deleted?

**Reply by adamhannigan** [reply]
Thanks for raising these. 

Again, there is no "privilege" to any farm. The bans occur from automatic bot detection. The team knows my stance on botting - even if my family breached the rules, they would be banned 🔥

4 mods have been banned/removed for breaking the rules with this automatic detection. 

I just checked out 486 and it has not been picked up yet as botting/sniping from our detection. The 2 possibilities here are:

1. They are a real person using tools to detect incoming trades (such as https://sfl.tools/listings) and then going to Goblin Village and buying these. These tools are fine, as long as a human is the one visiting goblin village and buying the item.
2. They are using advanced botting techniques that are not yet being discovered.

The trade sniping antibotting is still new (2 weeks old) and we are constantly improving it. If you could provide some more details that would help us detect these "snipers" it would be greatly appreciated 🙏 We need to have hard proof before banning anyone (i.e. prove they are not a human buying the trades)

Could you also provide some details around 26, 100025 and 144826 and we will look into it. Thanks!

**Comment by qiongloo** [comment]
Also, they seem to be very strict about selling low and high prices in the internal trading market, if you want to solve this problem set a minimum and maximum price for the item instead of talking about it in discord. I want to trade resources privately with friends at low or high prices. Isn't this transaction normal? Every time I see such inaction, I am very sad, and the rules do not say that it is not allowed.

**Comment by YouQunFang** [comment]
we need manager who is diamond hand of sunflower land, we do not need some one silly, i mean Labochi, I apologize i say that, but i can't stand it. ur team have to Verify funds of manager. did they cheat team.

**Reply by CelestineZoe** [reply]
You shouldn't slander others without seeing the facts clearly, you are too unreasonable

**Reply by YouQunFang** [reply]
anyway Announced assets of them

**Reply by YouQunFang** [reply]
check farm no26, why this guy can have power of life and death, he has nothing in sunflowerland project, why i have to explain to him?

**Comment by CelestineZoe** [comment]
This vote is not fair at all, with obvious personal subjective intentions, deliberately leading people to choose the second option, I will not participate

**Reply by masterYl** [reply]
I prefer the project side to come up with more specific options for us to choose instead of me

**Reply by 1176517584** [reply]
You can't experience this feeling without being a victim. When you play an account normally and invest a lot of money and energy, suddenly it is blocked, and then you understand for various reasons not to unlock it

**Comment by 17ss** [comment]
I predict privilege farm will transfer NFTs soon and an empty farm is banned

**Comment by Edward4396** [comment]
The SFL has reached 0.05, and two months ago there were almost 300 audiences in an AMA,  but 50 at most recently. The data speaks for everything. SFL is dying with no efforts.
BTW, Steve is incapable, he should blame for 90% of this situation.

**Comment by shuangyu1020a** [comment]
Don't freeze users' assets easily, because this will cause panic, you will never dare to buy NFTs and put them in the sunflower paradise

**Comment by adamhannigan** [comment]
Thanks for voicing your concerns everyone. Some great points around decentralisation, anti-botting and the future of the game.

When raising a discussion like this could you please provide your farm IDs and your family's farm IDs so the team can investigate. The bans are done automatically by our bot detection system which looks for patterns in on-chain forensics, IP/device conditions, gameplay activity, scripting software and more. We don't do manual bans, so in this case it is a bit difficult to tell under what circumstance you were banned without your farm ID.

As we mentioned in Discord, last week there was 2000+ automatic bans as a botter tried to infiltrate/drain SFL from our game. Unfortunately they created hundreds of fake Discord/Twitter accounts so it has been hard for us to go through requests from people who believe they were not multiaccounting/botting. Please be patient while we investigate your case 🙏 

We've been extremely clear on our multiple account approach. A player can own multiple NFT farms, but they can only play on one at a time. This ensures that the economy stays strong and one person does not get an advantage over everyone else in the community. If you were not playing/accessing multiple accounts at the same time, you should not have anything to worry about.

Labochi had a personal situation which led to the sale of the farm. They asked for permission 2 weeks in advance before they sold the farm and then waited an extra 2-3 weeks before they were given a new one from the team (test wallet). Let's try to focus on the discussion at hand around botting and the recent bans - we will be able to cooperate and move forward easier if we focus on this without calling people out.

As all of our moderators know, we have banned at least 4 people who were previously Sunflower Land moderators for playing on multiple accounts. There is no preferential treatment, nor will there be. Please do not involve them in these discussions - they are working hard for this community and trying to protect it. 

If you could please provide your farm ID and any details that can help us resolve the situation (i.e. why it may of been incorrectly detected as multi-accounting) could you please provide them. The team will look at it right away. Thanks!

**Reply by libbiashyra** [reply]
is there exist privilege farm? are they mutil-account?(26, 486, 100025, 144826)
https://github.com/sunflower-land/sunflower-land/discussions/1358#discussioncomment-3444952
is the picture real?

**Reply by TheDcol** [reply]
Mate, just stop..

![image](https://user-images.githubusercontent.com/108110065/186052929-8875356f-80e0-404c-87c7-3e3a25843a20.png)

**Reply by masterYl** [reply]
Great, glad you can answer most of my questions, adam, I always just wanted a decent explanation, if steven and Labochi were half as capable and worded as you, maybe the sunflower community would be better, At least things caused by me won't be so bad。


First of all, in the help channel, I sent my farm ID, wallet address, and my most loyal address to play, and also sent the address and wallet address to prove that we are real people, farm address：

- Farm Number:​89724  
- Farm Number:144076

And your discord operation team feels very mechanically that we are one person, simply relying on data to judge multiple numbers, look at this farm id:, I taught them to understand web3 and your game a few months ago, step by step Step by step teach him how to operate, from registering, playing, to using opensea
(Of course not only him, but also my other friends, all of them have separate IPs, devices, thousands of kilometers away from me), chat records can be queried, I'm afraid you don't understand Chinese, the following picture has been translated:
![下载 (2)](https://user-images.githubusercontent.com/50456562/186163353-75e100ab-8ae1-4233-bf67-0ff5d3acfb52.png)
![下载](https://user-images.githubusercontent.com/50456562/186163380-c5b536b7-d23c-46e2-8ac8-dc71d2c2816d.png)

We just have a collection of resources, and when your internal market has not been announced, we cannot make nft through internal methods, we can only rely on external transactions, and we follow the rules in the following figure:
![IMG_1990(20220823-202659)](https://user-images.githubusercontent.com/50456562/186163636-82b411b6-d0ec-432c-b54c-4b729217a483.JPG)

Does this violate the one-person-one-account rule?
As for what you said, you have blocked a number of loyalists who violated the rules, and will not protect any players. I also think that you have done a great job. Of course, I don't need you to treat them differently, and my situation is completely different from them. I am not a multi-number. Players, let me stress again, I recommend my family and friends to play the game, I give them money, it's not a one-man operation at all, they have complete freedom to manipulate their farm

Or as I said at the beginning, I have always just wanted an open, reasonable, positive explanation, not ambiguous, hegemonic, otherwise the protest will not end:

①If you think I am a multi-number player, please show me the evidence that I hold the same device number, ip, etc., instead of relying on your anti-cheating system, otherwise, please unblock the land of me and my friend (the above id)

②Can you explain what the person above said: Privileged Farm ID: 26, 486, 100025, 144826

③Please help other players in the Help channel who worked hard for Sunflower and were inexplicably banned, a saying

To put it simply, web3 is a kind of progress, not a bonus, and your operating practices, to be honest, are going backwards, which sounds a bit ugly (far worse than traditional games, see how traditional games do economic systems and how to deal with them). Studio, how to combat script behavior, how to communicate with players), just want me to support the projects I love, see the shortcomings and do better, and welcome a better web3

**Reply by adamhannigan** [reply]
Thanks @libbiashyra I missed that comment. I have just responded above - if you have any more details around these cases please help out and provide some evidence so we can ban them 🔨

**Reply by adamhannigan** [reply]
@masterYl Thanks for the details! I have been a bit out of loop with the focus on Land Expansion.

What you described is not a breach of the rules. Sunflower Land is a community game and we encourage people to work together and share assets. I myself had to help my brother start his farming adventure so I understand the need to send SFL and items to help them get started.

1. We are investigating this today and will keep you posted 🙏 From what it looks like is your account got caught in the middle of 16 player bot operation (that's what the algorithm detected) - we will double check the report to figure out why it picked up these 2 accounts.
2. I've just posted above. The sniping anti-botting is new and we are still improving it - if caught they will be banned 🔥 
3. Will do - we're understaffed and under a lot of pressure at the moment so things are just taking a while

Thanks for your patience

**Reply by masterYl** [reply]
Yes, I'm glad you answered my questions and reassured me that you can make sunflowers well.

In the Chinese community, many people (including robots) have had transactions with me, such as the following (I deleted the chat record, this is the chat record I found him to give me)
![下载 (6)](https://user-images.githubusercontent.com/50456562/186305690-709b8015-ea5b-4292-8572-109b2d23b22e.png)
![下载 (7)](https://user-images.githubusercontent.com/50456562/186305719-b0858893-ede6-43a6-ba44-a9519b150f89.png)

To be honest, I really don't know him, only on the Internet (but he was in financial difficulties at that time, so I lent him SFL.I also borrowed some RMB to him）, I collected a lot of their resources, and later their account was blocked. blocked, and implicated me

I hope that through this incident, you can improve your anti-script system, this time a lot of innocent people have been implicated and left because of it. (As you said you have been busy lately and ignored it, but I am in the community and indeed a lot of people leave)
And what makes me angry is that your operators believe too much in this anti-machine system, and their responses are also like a machine, blindly enforcing the rules, ignoring why the rules exist

Find the problem and fix him, then Sunflower will do better

**Comment by qiongloo** [comment]
4. Isn't it normal that VIP players are willing to invest money to do rare NFTs? As the number of people increases, 300 NFTs will be robbed all at once, which is normal, because resources can be traded and aggregated. Unless you ask for the resources of the seeds you mine when you mintNFT. Officials and other players seem to care about this, but in the current system, it's inevitable. 
5. VIP players went to op to buy a lot of nft, invested in land, and were blocked. This asset cannot be protected. Is it a big problem? Think about how you would feel if it were you. This has caused panic, and everyone’s money has been hard-earned. Originally, playing games and investing money would have a better experience, but now I am worried, which is very bad for the game.

**Reply by adamhannigan** [reply]
Thanks for your feedback.

It may be a communication/translation issue but could you please explain "300 NFTs will be robbed all at once". I don't understand what you mean here.

Players that buy assets are a critical part of our ecosystem and we build our infrastructure and game to support them as much as possible. We've done strenuous testing to ensure that anyone purchasing assets (i.e. off Open Sea) is not picked up as multiaccounting.

The "ban algorithm" allows players/traders to freely play the game. It looks for known clear bot patterns on the Blockchain to ensure that bad actors do not destroy the ecosystem and crash the price by draining SFL.

Usually these accounts consist of very little SFL and items. The botters/mulitaccounters drain the SFL as quickly as they can.

It is extremely important we do not ban a loyal player or someone that has invested into the game. When we do encounter an account with lots of assets that has been banned, it goes through a rigorous process of manually checking and confirming whether they were botting or multiaccounting (unfortunately this takes time)

**Comment by adamhannigan** [comment]
Thanks everyone for the feedback. It truly does improve our processes and our anti-bot system 🙏 

@masterYl The details you provided were extremely helpful. It appears the accounts you mentioned go caught up in multi-accounting bot detection which recognised patterns that were similar to multi-accounters (sending resources between accounts). Further investigate appears this was done following the game rules without automation or a root wallet connection.

The IP addresses + devices have confirmed it.

The accounts you listed have been unbanned ✅ We appreciate your patience.

Going forward we will include more checks around IP address/device recognition to ensure that innocent people do not get detected.

Also thanks to others who posted examples of accounts that may be breaching trading/botting rules. These are currently being investigated by the team. A big takeaway is that there is no privilege - we are working daily on our automatic anti-botting solution and it has no mercy 😅
